source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1514,11 +1514,31 @@ servers. For a "node is recovering" error, single-threaded clients MUST NOT
 mark the topology as "stale". If a node is recovering for some time, an
 immediate scan may not gain useful information.
 
-The client SHOULD clear its connection pool for the server.
+The following subset of "not master" errors is defined to be "server shutting
+down" errors:
+
+.. list-table::
+  :header-rows: 1
+
+  * - Error Name
+    - Error Code
+  * - InterruptedAtShutdown
+    - 11600
+  * - ShutdownInProgress
+    - 91
+
+If the client is connected to server version 4.2 or higher, and the client
+receives a "not master" error which is not a "server shutting down" error,
```

Comment:
`... receives a "not master" or "node is recovering" error which is not a "server shutting down" error, ...`

Reply:
Made this change, thank you.